### ISSUE TITLE
Unknown return code rowGetNNonz

### ISSUE THREAD

**Issue by stefanvanberkum** [scipopt/PySCIPOpt#591]
Hi all,

For a cut selector plugin, I am trying to compute the integral support for cuts. For this purpose, given a cut (a row) I computed: 
```self.model.getRowNumIntCols(cut) / self.model.rowGetNNonz(cut)```

However, this yields the following error:
```Traceback (most recent call last):
  File "/home/stefan/.local/share/JetBrains/Toolbox/apps/PyCharm-C/ch-0/221.5080.212/plugins/python-ce/helpers/pydev/_pydevd_bundle/pydevd_exec2.py", line 3, in Exec
    exec(exp, global_vars, local_vars)
  File "<input>", line 1, in <module>
  File "src/pyscipopt/scip.pyx", line 1934, in pyscipopt.scip.Model.rowGetNNonz
  File "src/pyscipopt/scip.pyx", line 271, in pyscipopt.scip.PY_SCIP_CALL
Exception: SCIP: unknown return code!
```

So, the issue seems to be with the PY_SCIP_CALL of ```self.model.rowGetNNonz(cut)```, as its counterpart ```cut.getNNonz()``` does work as expected. While it does not create any problems for me, I figured I'd let you know. I am running PySCIPOpt 4.2.0 (installed via Conda as recommended) with Python 3.9.

**Comment by Opt-Mucca** [comment]
Cheers for finding this error and using the cut selector! The error should be fixed by #592

**Comment by stefanvanberkum** [comment]
Thank _you_ for including it, the plugin will be very useful for my research :)! I'll close the issue then.
